guides/05-validators.md

```diff
@@ -199,10 +199,9 @@ If-None-Match: "0267aa812d66aafb7c4ffb790d8b5ffc"
 Host: example.com
 ```
 
-At this point, the server can do whatever it takes to revalidate determine the
-freshness of the cached content. For example, the server can re-apply the same
-hash function to the document and compare that against the `ETag` supplied by
-the client.
```

Comment:
`At this point, the server can do whatever it takes to revalidate determine the freshness of...` becomes `At this point, the server can do whatever it takes to determine the freshness of...`